src/catalog/zone_map_catalog.cpp

```diff
@@ -78,56 +82,88 @@ bool ZoneMapCatalog::InsertColumnStatistics(
   return return_val;
 }
 
-bool ZoneMapCatalog::DeleteColumnStatistics(oid_t database_id, oid_t table_id,
-                                            oid_t tile_group_id,
-                                            oid_t column_id,
-                                            concurrency::TransactionContext *txn) {
+bool ZoneMapCatalog::DeleteColumnStatistics(
+    oid_t database_id, oid_t table_id, oid_t tile_group_id, oid_t column_id,
+    concurrency::TransactionContext *txn) {
   oid_t index_offset = static_cast<int>(IndexId::SECONDARY_KEY_0);
-  std::vector<type::Value> values({
-    type::ValueFactory::GetIntegerValue(database_id),
-    type::ValueFactory::GetIntegerValue(table_id),
-    type::ValueFactory::GetIntegerValue(tile_group_id),
-    type::ValueFactory::GetIntegerValue(column_id)
-  });
+  std::vector<type::Value> values(
+      {type::ValueFactory::GetIntegerValue(database_id),
+       type::ValueFactory::GetIntegerValue(table_id),
+       type::ValueFactory::GetIntegerValue(tile_group_id),
+       type::ValueFactory::GetIntegerValue(column_id)});
   return DeleteWithIndexScan(index_offset, values, txn);
 }
 
 std::unique_ptr<std::vector<type::Value>> ZoneMapCatalog::GetColumnStatistics(
     oid_t database_id, oid_t table_id, oid_t tile_group_id, oid_t column_id,
     concurrency::TransactionContext *txn) {
-  std::vector<oid_t> column_ids(
-      {static_cast<int>(ColumnId::MINIMUM), 
-       static_cast<int>(ColumnId::MAXIMUM), 
-       static_cast<int>(ColumnId::TYPE)});
-
-  oid_t index_offset = static_cast<int>(IndexId::SECONDARY_KEY_0);
-
-  std::vector<type::Value> values({
-    type::ValueFactory::GetIntegerValue(database_id),
-    type::ValueFactory::GetIntegerValue(table_id),
-    type::ValueFactory::GetIntegerValue(tile_group_id),
-    type::ValueFactory::GetIntegerValue(column_id)
-  });
-
-  auto result_tiles =
-      GetResultWithIndexScan(column_ids, index_offset, values, txn);
-
-  PELOTON_ASSERT(result_tiles->size() <= 1);  // unique
-  if (result_tiles->size() == 0) {
+  std::vector<oid_t> column_ids({static_cast<int>(ColumnId::MINIMUM),
```

Comment:
You might want to add more tests to cover all the modified code paths and increase your code coverage